pleskdistup/actions/grub.py

```diff
@@ -0,0 +1,27 @@
+# Copyright 2023-2024. WebPros International GmbH. All rights reserved.
+import os.path
+
+from pleskdistup.common import action, dist, util
+
+DEBCONF_CMD = "/usr/bin/debconf-show"
+
+class AssertGrubInstallDeviceExists(action.CheckAction):
+    def __init__(self) -> None:
+        self.name = "check GRUB installation device exists"
+        self.description = """Grub's install-device is not found: {}
+This could fail conversion - please fix it.
+"""
+
+    def _do_check(self) -> bool:
+        if not dist.get_distro().deb_based:
+            return True
+
+        debconf_out = util.logged_check_call([DEBCONF_CMD, "grub-pc"])
+        prefix = "grub-pc/install_devices:"
+        for line in debconf_out.split("\n"):
+            if line.find(prefix) == 0:
+                devpath = line[len(prefix):].strip()
+                if not os.path.exists(devpath):
```

Comment:
Is it possible to have multiple `grub-pc/install_devices:` lines in the output?

Reply:
The virtual instance I encountered had only one device path. Although the plural noun suspects of multi device but I could not find a way to make it.